examples/ivr_outbound_dialer.py

```diff
@@ -0,0 +1,374 @@
+#
+#Sample Switchy Applications
+#  https://switchy.readthedocs.org/en/latest/apps.html
+#
+#License:
+#  BSD License
+#  http://opensource.org/licenses/bsd-license.php
+#
+#  Copyright (c) 2015, Sangoma Technologies Inc
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without modification, 
+#  are permitted provided that the following conditions are met:
+#  1. Developer makes use of Sangoma NetBorder Gateway or Sangoma Session Border Controller
+#  2. Redistributions of source code must retain the above copyright notice, 
+#     this list of conditions and the following disclaimer.
+#  3. Redistributions in binary form must reproduce the above copyright notice, 
+#     this list of conditions and the following disclaimer in the documentation 
+#     and/or other materials provided with the distribution.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS 
+#  OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF 
+#  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. 
+#  IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, 
+#  INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+#  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, 
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE 
+#  OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF 
+#  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+import time
+import switchy
+import threading
+from switchy.marks import event_callback
+from switchy.apps.players import TonePlay
+from switchy import get_originator
+
+# Enable logging to stderr
+log = switchy.utils.log_to_stderr()
+# Debug levels: INFO in production, DEBUG in devel
+log.setLevel("INFO")
+
+# Specify FreeSWITCH or Sangoma NSG IP information
+# In this example the sample app is running on 
+# Sangoma NSG appliance hence the use of local address
+host = "127.0.0.1"
+port = 8821
+
+
+# IVR Call Logic
+# This is the main IVR logic
+# All custom development should be done here.
+# Variables
+#   self.<variables>       are global in nature.
+#   call.vars.['var_name'] should be used for per call info and state
+#
+class IVRCallLogic(object):
+
+    def prepost(self, client, listener):
+        # Get the install directory of FreeSWITCH or Sangoma NSG and append recording to it
+        # By default FreeSWITCH resides in /usr/local/freeswitch, NSG resides in /usr/local/nsg
+        self.recdir = client.cmd('global_getvar base_dir')
+        self.recdir += "/recording"
+        log.info("Setting recording dir to '{}" . format(self.recdir))
+        
+        # Get the install directory of NSG and append sounds to it
+        self.sound_dir = client.cmd('global_getvar base_dir')
```

Comment:
done.

NSG global variables are bit screwed up do to building in a sandbox.
so sound directory points to a wrong path :)

Nenad